openslides/utils/cache_providers.py

```diff
@@ -113,22 +130,28 @@ class RedisCacheProvider(BaseCacheProvider):
     def __init__(self, redis: str) -> None:
         self.redis_address = redis
 
-    async def get_connection(self) -> 'aioredis.RedisConnection':
+    def get_connection(self) -> RedisConnectionContextManager:
+        """
+        Returns contextmanager for a redis connection.
+        """
+        return RedisConnectionContextManager(self.redis_address)
+
+    async def clear_cache(self) -> None:
         """
-        Returns a redis connection.
+        Deleted all cache entries created with this element cache.
         """
-        if self.redis_pool is None:
-            self.redis_pool = await aioredis.create_redis_pool(self.redis_address)
-        return self.redis_pool
+        async with self.get_connection() as redis:
+            # TODO: Fix me. Do only delete keys, that are created with this cache.
+            await redis.flushall()
```

Comment:
Ah, this should be fixed as soon as possible because developers may flush their redis instance accidentally. Please open an urgent issue for this.

Reply:
It is not possible to raise a warning because it would be printed many times in the tests and also in migration.

Reply:
I disabled the test with redis code. So currently the problematic function is not used anywhere. I will fix this in another pr

Reply:
Ok.